pkg/executor/distsql.go

```diff
@@ -620,14 +631,14 @@ func (e *IndexLookUpExecutor) open(_ context.Context) error {
 
 	var err error
 	if e.corColInIdxSide {
-		e.dagPB.Executors, err = builder.ConstructListBasedDistExec(e.buildPBCtx, e.idxPlans)
+		e.dagPB.Executors, err = builder.ConstructListBasedDistExec(e.buildPBCtx, e.idxPlans, e.storeType)
```

Comment:
Since TiFlash support MPP, TiDB always generate tree based plan for TiFlash, I suggest we keep this assumption.

Reply:
addressed.